src/OpenTelemetry.Instrumentation.GrpcCore/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.0.0-beta.6
+
```

Comment:
It will be great to put here also previous date, but we can just start tracking from this version.

```suggestion

Released 2024-May-29

```